### ISSUE TITLE
[Bug][DropDownList] aria errors on open state

### ISSUE THREAD

**Issue by iadnonov** [telerik/kendo-react#1797]
## I'm submitting a...

* Bug report 

## Current behavior

Testing the DropDownList with axe DevTools generate the following errors:

- missing `aria-controls`
- wrong `aria-activedescendant` value

## Minimal reproduction of the problem with instructions

Test opened DropDownList with axe DevTools

## Environment

Package versions:

```
7.0.1
```

Can be reproduced here: https://stackblitz.com/edit/react-hfmscx-pp9a6m?file=app%2Fmain.jsx

Reported in Ticket ID: 1526805
